webknossos/webknossos/dataset/layer.py

```diff
@@ -1092,18 +1187,21 @@ def _setup_mag(self, mag: Mag, path: Optional[str] = None) -> None:
         mag_name = mag.to_layer_name()
 
         self._assert_mag_does_not_exist_yet(mag)
-
+        mag_path_maybe = UPath(path) if path else path
         try:
             cls_array = BaseArray.get_class(self._properties.data_format)
-            info = cls_array.open(
-                _find_mag_path_on_disk(self.dataset.path, self.name, mag_name, path)
-            ).info
```

Comment:
This is the only code line where the fourth argument is passed to `_find_mag_path_on_disk`. See my comment below.

My problem is that I do not know the code base well enough to foresee a scenario where my changes might break the code.